braindecode/augmentation/transform_class.py

```diff
@@ -0,0 +1,26 @@
+from functools import partial
+import random
+
+
+class Transform:
+
+    def __init__(self, operation, probability=None, magnitude=0):
+        self.operation = operation
+        self.probability = probability
+        self.magnitude = magnitude
+        self.transform = partial(operation, magnitude=magnitude)
```

Comment:
I would not make the assumption that any operation must have some magnitude.

Reply:
Rather just don't have magnitude and if your operation needs it you can still supply it directly to your transform object or via partial to your transform function if needed

Reply:
Rather just don't have magnitude and if your operation needs it you can still supply it directly to your transform object or via partial to your transform function if needed

Reply:
+1